### PR TITLE
Documentation changes: README.md

### PR DESCRIPTION
**Pull request by mattfenwick**
I changed 4 things: one in each commit of this PR.  IMO, each of these changes make the Readme a bit easier to read.  

Please let me know if there's anything wrong with this PR, or if there's anything I can do to improve it!  For example, if you like some of the changes but not others, I'd be happy to create a new PR with just the good changes.

Thanks for taking the time to look at this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,6 +91,12 @@ future {
 While this is really short and simple, it is equally limited. In many cases, you will need a way to indicate that the task failed. To do this, instead of returning the value, you can return a Result. Results can indicate either a success or a failure:
 
 ```swift
+enum ReadmeError: ErrorType {
+    case
+    RequestFailed,
+    TimeServiceError
```
Comment (Thomvis) [inline README.md]: Could you update the enum definition to the common syntax. The [Swift book](https://developer.apple.com/library/ios/documentation/Swift/Conceptual/Swift_Programming_Language/Enumerations.html#//apple_ref/doc/uid/TP40014097-CH12-ID146) uses either a single line with all cases prefixed by `case` or one case per line with `case` on every line.

**Comment by Thomvis** [comment]
Thanks for the PR! These are nice improvements and I hope to merge them in soon. I added one comment that I hope you're able to address first.

Thanks again!

File: README.md (hunk 1 above)
Comment (mattfenwick) [inline README.md]: I just made that change -- thanks for taking a look!

**Comment by Thomvis** [comment]
💯, thanks!

**Pull request merged**